site/layouts/partials/social.html

```diff
@@ -8,12 +8,12 @@
   {{- $socialImagePath = urls.JoinPath $socialImagePath "brand/bootstrap-social.png" -}}
 {{- end -}}
 
-<meta name="twitter:card" content="summary_large_image">
-<meta name="twitter:site" content="@{{ .Site.Params.twitter }}">
-<meta name="twitter:creator" content="@{{ .Site.Params.twitter }}">
-<meta name="twitter:title" content="{{ $pageTitle }}">
-<meta name="twitter:description" content="{{ $pageDescription }}">
-<meta name="twitter:image" content="{{ $socialImagePath | absURL }}">
+<meta name="x:card" content="summary_large_image">
```

Comment:
just wondering... are the changes made here for the partial confirmed as working? The latest docs still say to use 'twitter:card', not 'x:card' for example. https://developer.x.com/en/docs/twitter-for-websites/cards/overview/markup

Reply:
I agree. `<meta>` should stay with `twitter:*`.

Reply:
Thanks for raising it, it should be good now.